src/components/map/MapFilter.js

```diff
@@ -24,14 +24,22 @@ const MapFilter = () => {
 
   const dispatch = useDispatch();
 
-  const [searchInput, setSearchInput] = useState('');
+  const [citySearchInput, setCitySearchInput] = useState('');
+  const [stateSearchInput, setStateSearchInput] = useState('');
 
-  const handleSearchInput = e => {
-    setSearchInput(e.target.value.toLowerCase());
+  const handleCitySearchInput = e => {
+    setCitySearchInput(e.target.value.toLowerCase());
+  };
+
+  const handleStateSearchInput = e => {
```

Comment:
For this component, does it make more sense to have this to be a drop down instead of a text input?

Reply:
I agree that a drop down would be preferable. I'm thinking given the time constraints we are under and other priorities, the functionality as it currently stands is a value add for the user. If we have time I can circle back and change this to a drop down.